benchmarks/runner.py

```diff
@@ -101,6 +103,11 @@
     "devices": [
         "cuda",
     ],
+    "quick": {
```

Comment:
i think it makes sense to offer this 'quick' thing.  but it doesn't necessarily make sense to pick 2 resnets here.  For example, why not pick the handful of models known to have either high peak mem or compile times?

Reply:
I actually never intended for actual run. This was just to speedup development process, so that we can quickly run the runner and look at graphs etc. I dont want to add more models here.

I think we can add another flag, that does what you say. There, we could have handful of models (and not just 2), which represents the broader set, and can finish in an hour or so.